packages/salesforcedx-webview-ui/package.json

```diff
@@ -0,0 +1,118 @@
+{
+  "name": "@salesforce/salesforcedx-webview-ui",
+  "description": "UI components for Salesforce Extensions for VS Code",
+  "version": "42.18.0",
+  "publisher": "salesforce",
+  "license": "BSD-3-Clause",
+  "private": true,
+  "engines": {
+    "vscode": "^1.17.0"
+  },
+  "dependencies": {
```

Comment:
Everything webpack, loaders, jest etc should be under devDependencies.

Reply:
Interesting. I got this from create-react-app. This was the section that they put it in. I can move them into dev-dependencies.

Reply:
I guess it depends on how we are bundling the extension, are we keeping them as part of node_modules when creating the extension?

Reply:
Nope, we only bundle the stuff that is optimized in the /build folder. So nothing from node_modules leaks in. Everything has to be bundled in a way that you can just load the .html/.js/etc. All the JS is bundled as UMD, not That's why I think it's OK and that is why create-react-app did it that way. I will investigate more but I don't think I change this from how they were doing it.

Reply:
I'm looking at https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/package.json and https://github.com/wmonk/create-react-app-typescript/blob/master/packages/react-scripts/package.json 

It seems that react, react-dom, and typescript are in the devDependencies. Let me move those back in there to follow that (not sure how it got moved into dependencies).

I will keep webpack and rest in dependencies for now since that seems to be how the template it.

Reply:
More info at https://github.com/facebook/create-react-app/issues/1764